docs/modules/SDDs/pages/0006-commodore-configuration-catalog-compilation.adoc

```diff
@@ -82,33 +82,33 @@ Component versions can be any git https://git-scm.com/docs/gitglossary#Documenta
 
 * Components are versioned as part of the inventory repositories, which comes from VSHN defaults and the cluster / customer specific repository
 * Each component is managed in an individual Git repository, comparable to a Puppet module.
-* Components bundle the Kapitan component templates (for example Jsonnet) with the component class that will be referenced in the Kapitan inventory and optional postprocessing filters which are applied on the compiled Kapitan catalog. The engine downloads the component repository into dependencies  and symlinks the component class into inventory/classes/components  to make the class available in the Kapitan inventory structure.
+* Components bundle the Kapitan component templates (for example Jsonnet) with the component class that will be referenced in the Kapitan inventory and optional postprocessing filters which are applied on the compiled Kapitan catalog. The engine downloads the component repository into `dependencies/` and symlinks the component class into `inventory/classes/components` to make the class available in the Kapitan inventory structure.
 * A component repository must have the file `class/<component-name>.yml` which contains the Kapitan component class which defines how the component is compiled
-* A component repository must define default parameters for the templates in the file `class/defaults.yml` in order to allow components to be included at any level in the configuration hierarchy without overriding component parameters which have been configured in the hierarchy earlier than the component was included. The `defaults.yml`  file of a component is symlinked to `inventory/classes/defaults/<compoent-name>.yml`  and included in the Kapitan target before `global.common` .
-* If a component has postprocessing filters, it must have the file `postprocess/filters.yml`  defines which compilation engine postprocessing filters should be applied.
+* A component repository must define default parameters for the templates in the file `class/defaults.yml` in order to allow components to be included at any level in the configuration hierarchy without overriding component parameters which have been configured in the hierarchy earlier than the component was included. The `defaults.yml` file of a component is symlinked to `inventory/classes/defaults/<compoent-name>.yml` and included in the Kapitan target before `global.common`.
+* If a component has postprocessing filters, it must have the file `postprocess/filters.yml` to define which compilation engine postprocessing filters should be applied.
```

Comment:
Probably better "to define which postprocessing filters the compilation engine should apply".